codewof/templates/users/group_detail.html

```diff
@@ -0,0 +1,205 @@
+             href="{% url 'users:groups-memberships-invite' group.pk %}">Add Members</a>
+          {% endif %}
+          <h2>Members</h2>
+
+          <table id="members-table" class="table">
+            <thead>
+            <tr>
+              <th scope="col">Email</th>
+              <th scope="col">Name</th>
+              <th scope="col">Role</th>
+              {% if is_admin %}
+              <th scope="col">Remove?</th>
+              {% endif %}
+            </tr>
+            </thead>
+
+            <tbody id="members-table-tbody">
+            {% for membership in memberships %}
+            <tr id="membership-{{ membership.pk }}">
+              <td>{{ membership.user.email }}</td>
+              <td>{{ membership.user.first_name }} {{ membership.user.last_name }}</td>
+              {% if is_admin %}
+              <td>
+                <select id="membership-{{ membership.pk }}-select" class="form-select"
+                        aria-label="Default select example" autocomplete=off>
+                  {% for role in roles %}
+                  {% if role == membership.role %}
+                  <option selected>{{ role }}</option>
+                  {% else %}
+                  <option>{{ role }}</option>
+                  {% endif %}}
+                  {% endfor %}
+                </select>
+              </td>
+              <td class="text-center">
+                <input id="membership-{{ membership.pk }}-checkbox" class="form-check-input" type="checkbox" value=""
+                       autocomplete=off {% if user.pk == membership.user.pk %}disabled{% endif %}>
+              </td>
+              {% else %}
+              <td>{{ membership.role }}</td>
+              {% endif %}
+            </tr>
+            {% endfor %}
+            </tbody>
+
+          </table>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col mb-3">
+          <a id="email-button" class="btn btn-primary mb-1 btn-sm float-right" href="">Email All</a>
+          {% if is_admin %}
+          <button id="save-button" class="btn btn-success btn-sm mb-1">Save</button>
+          {% endif %}
+        </div>
+      </div>
+    </div>
+
+    {% if feed %}
+    <div class="col-12 col-md-4 mb-3">
+      <h2>Feed</h2>
+
+      <table class="table">
+        <tbody id="feed-table-tbody">
+        <script>let attemptLikeNames = {}</script>
+        <script>let likeNames = []</script>
+        {% for attempt in feed %}
+        {% get_attempt_like_users_for_group attempt group.pk as like_users %}
+        <tr id="attempt-{{ attempt.pk }}">
+          <td class="feed-text">{{ attempt.profile.user.first_name }} {{ attempt.profile.user.last_name }} answered
+            Question <a href="{% url 'programming:question' attempt.question.pk %}">{{ attempt.question.title }}</a>
+            ({{ attempt.datetime|localtime }})
+          </td>
+          <td class="align-middle td-thumb">
```

Comment:
Not too sure. My reasoning is that in english you read left to right, so that a feed entry would be interpreted as "Jackie Qiu answered blah which was liked by X people", whereas I don't think it would be as natural semantically the other way around.

Reply:
Logical reasoning, see what happens in your UI study.